plugins/cookie_manager/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+# 2.1.1
```

Comment:
And the pubspec. :)

Reply:
> And the pubspec. :)

thanks for remind, pubspec update already.